crates/lib3h/src/real_engine.rs

```diff
@@ -41,11 +43,25 @@ pub struct RealEngine<T: Transport> {
     transport_gateway: P2pGateway<T, RrDht>,
     /// Map of P2p gateway per Space+Agent
     space_gateway_map: HashMap<PlayerId, P2pGateway<TransportSpace, RrDht>>,
+    /// Our TransportId (aka MachineId, "HcMyadayada")
+    _transport_id: String,
+    /// The TransportId public key
+    _transport_public_key: Vec<u8>,
+    /// The TransportId secret key
+    _transport_secret_key: SecBuf,
+    /// needed to accept the Crypto trait generic
+    _phantom_crypto: std::marker::PhantomData<Crypto>,
```

Comment:
Why the '_' prefixes here? Are these unused fields?

Reply:
Since the struct doesn't have any members that use the `Crytpo` trait (used in member functions), we need to have the phantom data marker to satisfy the compiler... but it's not really there, hence the `_`.

The others are temporarily `_` to satisfy the linter until we implement code that uses them.

Reply:
Using `_` on struct fields is quite broken and painful to use.
As an alternative, using `#[allow(dead_code)]` also makes the linter happy.

Reply:
I'm only ok with `allow(dead_code)` if it won't bleed over and allow other bits of dead code to sneak in.

Reply:
yeah, when no longer dead we need to be careful to remove it... and also, make sure it's as local as possible.

I went ahead and switched to it for now.